extensions/wikia/AdEngine3/src/bill-the-lizard-wrapper.js

```diff
@@ -1,11 +1,21 @@
 import {
+    BillTheLizard,
     billTheLizard,
+    billTheLizardEvents,
     context,
+    events,
+    eventService
 } from '@wikia/ad-engine';
 import targeting from './targeting';
+import pageTracker from './tracking/page-tracker';
+
+const baseSlotName = 'INCONTENT_BOXAD_1';
```

Comment:
Rename to `garfieldSlotsBidderAlias`? Now it's too general. Imagine we'll add different model - it will be hard to guess why we need this variable or should we include it somewhere.

Reply:
done